tests/RequestValidatorTest.php

```diff
@@ -433,6 +433,32 @@ public function allOfSchemaProvider()
             ],
         ];
     }
+
+    public function test_handles_query_parameters()
+    {
+        Spectator::using('Test.v1.json');
+
+        // When testing query parameters, they are not found nor checked by RequestValidator->validateParameters().
+        Route::get('/users', function () {
+            return [];
+        })->middleware(Middleware::class);
+
+        $this->getJson('/users?order=fart')
```

Comment:
@hotmeteor AFAIK the `order` parameter should be causing a failure since it doesn't not adhere to enum requirement in the spec; however, it is not being checked